### PR TITLE
Updating tools/diamond from version 2.0.8 to 2.0.15

### PR DESCRIPTION
**Pull request by planemo-autoupdate**
Hello! This is an automated update of the following tool: **tools/diamond**. I created this PR because I think the tool's main dependency is out of date, i.e. there is a newer version available through conda.

I have updated tools/diamond from version 2.0.8 to 2.0.13.

**Project home page:** https://github.com/bbuchfink/diamond/releases

### CODE REVIEW

**Comment by gallardoalba** [comment]
The PR includes the following modifications:

- New parameters: --itinerate, --algo, --global-ranking, --max-hsps, --sed-cut, --freq-masking, --motif-masking
- The --masking parameter has been modified in order to included the new option
- New tests
- The --taxonlist parameter has been fixed.
- New sensitive mode: --fast
- Improved help sections
- Two sections have been created: `advanced options` and `output options`.

BLAST database compatibility haven't been included, since one of the command required for using it (diamond prepdb) is not available in the source code.

File: tools/diamond/test-data/diamond_view_results.tabular [hunk 1]
```diff
@@ -1,2 +1,2 @@
-sequence	gi|5524211|gb|AAD44166.1|	99.6	284	0	1	1	283	1	284	1.44e-205	550	94M1D189M	100
```
Comment (bernt-matthias) [inline tools/diamond/test-data/diamond_view_results.tabular:1]: do you know why this changed? The column that is changing is still the bitscore, or?

Comment (gallardoalba) [reply tools/diamond/test-data/diamond_view_results.tabular:1]: It seems that the daa file doesn't codify correctly the bitscore field, but I need to do some additional research.

Comment (gallardoalba) [reply tools/diamond/test-data/diamond_view_results.tabular:1]: It has been fixed in the last update.

**Review by bgruening — APPROVED** [review]
Very nice!

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**